src/org/labkey/test/tests/ExpTest.java

```diff
@@ -160,29 +159,25 @@ public void testSteps()
         selectQuery("exp", "Data"); // Select the one we want to edit
         waitForElement(Locator.linkWithText("edit metadata"), WAIT_FOR_JAVASCRIPT); //on Ext panel
         clickAndWait(Locator.linkWithText("edit metadata"));
-        waitForElement(Locator.xpath("//span[contains(text(), 'Reset to Default')]"), defaultWaitForPage);
-        int lastFieldIndex = getElementCount(Locator.xpath("//input[starts-with(@name, 'ff_label')]")) - 1;
-        Locator lastField = Locator.xpath("//input[@name='ff_label" + lastFieldIndex + "']");
-        click(lastField);
-        click(Locator.xpath("//span").append(Locator.lkButton("Alias Field")));
-        selectOptionByText(Locator.name("sourceColumn"), "RowId");
```

Comment:
The test used to be selecting the "RowId" for the alias field. I don't see that being done explicitly in your updated test, is that because it is the first field and already selected?

Reply:
Correct.

Reply:
Cool. Maybe add a quick comment to that affect then (i.e. that the selected options is already RowId so no need to change).

Reply:
added